sentry/src/main/java/io/sentry/util/CollectionUtils.java

```diff
@@ -145,6 +145,22 @@ public static int size(final @NotNull Iterable<?> data) {
     return filteredList;
   }
 
+  /**
+   * Returns true if the element is present in the array, false otherwise.
+   *
+   * @param array - the array
+   * @param element - the element
+   * @return true if the element is present in the array, false otherwise.
+   */
+  public static <T> boolean contains(final @NotNull T[] array, final @NotNull T element) {
+    for (T t : array) {
+      if (t.equals(element)) {
```

Comment:
I guess in theory the element in the array can be null. So we could either guard it via `t != null && ...` or reverse the check (since element is not-null) `element.equals(t)`